disnake/ext/commands/params.py

```diff
@@ -320,7 +323,7 @@ async def get_default(self, inter: CommandInteraction) -> Any:
     async def verify_type(self, inter: CommandInteraction, argument: Any) -> Any:
         """Check if a type of an argument is correct and possibly fix it"""
         # these types never need to be verified
-        if self.discord_type.value in [3, 4, 5, 8, 9, 10]:
+        if self.discord_type.value in [3, 4, 5, 8, 9, 10, 11]:
```

Comment:
Maybe we should use a set here

Reply:
we should just get rid of this entire method tbh